stellargraph/layer/gcn_lstm.py

```diff
@@ -332,16 +281,69 @@ def __init__(
         self.bias_regularizer = regularizers.get(bias_regularizer)
         self.bias_constraint = constraints.get(bias_constraint)
 
-        self.gcn_1 = GraphConvolution(units = layer_sizes[0], A=adj, activation=self.activations[0])
-        self.gcn_2 = GraphConvolution(units = layer_sizes[0], A=adj, activation=self.activations[1])
-        self.lstm = LSTM(self.layer_sizes[1], return_sequences=False)
-        self.dense = Dense(self.outputs, activation=self.activations[2])
+        # Activation function for each gcn layer
+        if gc_activations is None:
+            gc_activations = ["relu"] * n_gc_layers
+        elif len(gc_activations) != n_gc_layers:
+            raise ValueError(
+                "Invalid number of activations; require one function per graph convolution layer"
+            )
+        self.gc_activations = gc_activations
+
+        # Activation function for each gcn layer
+        if lstm_activations is None:
+            lstm_activations = ["tanh"] * n_lstm_layers
+        elif len(lstm_activations) != n_lstm_layers:
+            raise ValueError(
+                "Invalid number of activations; require one function per lstm layer"
+            )
+        self.lstm_activations = lstm_activations
+
+        self.gcn_1 = FixedAdjacencyGraphConvolution(
+            units=self.gc_layer_sizes[0], A=self.adj, activation=self.gc_activations[0]
+        )
+        self.gcn_2 = FixedAdjacencyGraphConvolution(
+            units=self.gc_layer_sizes[1], A=self.adj, activation=self.gc_activations[1]
+        )
+        self.lstm = LSTM(
```

Comment:
Same as my comment above, the docstring implies the user can specify any number of `LSTM` layers but this is hard coded to only use 1. It would be cool if we can specify any number of `LSTM` units!

```suggestion
        self.lstm_layers = [
            LSTM(units, act, return_sequences=False) for units, act in zip(self.lstm_layer_sizes, self.lstm_activations)
        ]
```